modules/core/client/config/core.client.menus.js

```diff
@@ -0,0 +1,48 @@
+(function () {
+  'use strict';
+
+  // Configuring the Fishing module
+  angular
+  .module('core')
+  .run(MenuConfig);
+
+  MenuConfig.$inject = ['Menus'];
+
+  function MenuConfig(Menus) {
+
+    Menus.addMenu('account', {
+      roles: ['user']
+    });
+
+    Menus.addMenuItem('account', {
+      title: '',
+      state: 'settings',
+      type: 'dropdown',
+      roles: ['user']
+    });
+
```

Comment:
Agreed the menu structure is odd.  Took me a while to find out that I had to do an addMenu.  Seems like too many levels.  I checked out the Bootstrap 4 docs and it looks fairly compatible, but I would wait until it drops before rewriting the Menu service:

http://v4-alpha.getbootstrap.com/components/navs/